clans/src/main/java/me/mykindos/betterpvp/clans/clans/listeners/ClanEventListener.java

```diff
@@ -62,17 +46,20 @@ public class ClanEventListener extends ClanListener {
     private final WorldBlockHandler blockHandler;
     private final Clans clans;
 
+    private final ClanCommand clanCommand;
```

Comment:
While this technically is more performant, it seems wrong inject this particular instance

I'd recommend injecting the CommandManager, and using getCommandByAlias to store or access the ClanCommand

Reply:
implemented